scope-validation-announcement.md

```diff
@@ -0,0 +1,10 @@
+# We are keep improving our service! But we might affect you , please read this.
+With the initial release of the ACS we allowed to request the scopes for the tokens. Please refer here for the [Access Token scopes](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-javascript)
+
+However we have not enforced the VoIP and PSTN scopes. Meaning you could make a call with  a chat only scope in your token.
```

Comment:
maybe rephrase as "...we have not enforced the 'voip' scope for voip or pstn calls. Meaning you could make a voip call with 'chat' only scope or pstn call with 'pstn' only scope. "